src/objects/feature-group/feature-group.stories.mdx

```diff
@@ -0,0 +1,38 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
+// See: https://github.com/webpack-contrib/raw-loader#examples
+// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
+// okay with the following Webpack-specific raw loader syntax. It's better to leave
+// the ESLint rule enabled globally, and only thoughtfully disable as needed (e.g.
+// within a Storybook docs page and not within an actual component).
+// This can be revisited in the future if Storybook no longer relies on Webpack.
+// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
+import defaultDemoSource from '!!raw-loader!./demo/demo.twig';
+import defaultDemo from './demo/demo.twig';
+
+<Meta title="Objects/Feature Group" />
```

Comment:
Since this pattern depends on media queries, you should add `parameters={{ docs: { inlineStories: false } }}` to this `Meta` element. This will cause the stories to display in an iframe in Docs view.